src/types.ts

```diff
@@ -223,3 +223,5 @@ export interface FilterRule {
 
 export const STORE_TYPES = ["credits", "marks"] as const
 export type StoreType = typeof STORE_TYPES[number]
+
+declare module browser {}
```

Comment:
globals should go into a different file but this should get replaced with the types library i linked above